cypress/e2e/hub/collections.cy.ts

```diff
@@ -222,9 +222,23 @@ describe('Collections Details View', () => {
 
   it.skip('user can deprecate a collection', () => {});
 
-  it.skip('user can delete version from system', () => {});
+  it('user can delete version from system', () => {
+    cy.approveCollection(collection, namespace, '1.0.0');
+    cy.galaxykit('task wait all');
+    cy.get('[data-cy="table-view"]').click();
+    cy.clickLink(collection);
+    cy.wait(400);
+    cy.selectDetailsPageKebabAction('delete-version-from-system');
```

Comment:
If you are doing some action in the test, its good to check if the action was successful - thus version is really missing in the system - otherwise we are only testing if the operation is not failing, but not if its doing what its expected. I saw this missing in many tests and it will need fine tuning.

And for testing version deletion, you have to upload and approve the collection with the same name and same namespace, but different version.

cy.galaxykit('collection upload namespace name version');

Then select one of those versions, delete it and check if the version is really missing.